<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>805</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="testing_tab_scroll_area">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>654</width>
        <height>785</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QFormLayout" name="test_model_info_layout">
         <item row="2" column="0">
          <widget class="QLabel" name="test_run_name_label">
           <property name="text">
            <string>Test run name</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="test_run_name"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="test_model_selection_label">
           <property name="text">
            <string>Model</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="test_model_selection"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Output raster</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QgsFileWidget" name="test_output_raster">
           <property name="storageMode">
            <enum>QgsFileWidget::SaveFile</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Model file</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="model_file_testing">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="test_evidence_data_box">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Evidence data</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <layout class="QVBoxLayout" name="test_evidence_data_layout"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="test_label_data_box">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Labels</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QgsMapLayerComboBox" name="test_label_data"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="metrics_box">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>500</height>
          </size>
         </property>
         <property name="title">
          <string>Metrics</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QStackedWidget" name="test_metrics_stack">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="classification_metrics_page">
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QCheckBox" name="accuracy_checkbox">
                <property name="text">
                 <string>Accuracy</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="precision_checkbox">
                <property name="text">
                 <string>Precision</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="recall_checkbox">
                <property name="text">
                 <string>Recall</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="f1_checkbox">
                <property name="text">
                 <string>F1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="auc_checkbox">
                <property name="text">
                 <string>AUC</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="regression_metrics_page">
             <layout class="QVBoxLayout" name="verticalLayout_22">
              <item>
               <widget class="QCheckBox" name="mse_checkbox">
                <property name="text">
                 <string>MSE</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="rmse_checkbox">
                <property name="text">
                 <string>RMSE</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="mae_checkbox">
                <property name="text">
                 <string>MAE</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="r2_checkbox">
                <property name="text">
                 <string>R2</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="reset_run_layout">
         <item>
          <widget class="QPushButton" name="test_reset_btn">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="test_run_btn">
           <property name="text">
            <string>Run</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Progress</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="training_progress_bar_3">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
